### ISSUE TITLE
Epub format?

### ISSUE THREAD

**Issue by ccamara** [djnavarro/rbook#28]
Thanks for writing and sharing this book, @djnavarro 

Would it be possible to generate a copy in `epub` format for reading on an e-book? AFAIK, bookdown allows it, but I have not managed to generate a file (never worked with it, though). Also, converting PDF to epub using calibre or pandoc does lose some formatting and makes it extremely difficult to read.

**Comment by djnavarro** [comment]
Yeah, this has been an issue from the beginning. The book was originally written in pure LaTeX and the bookdown conversion generously made by @ekothe doesn't quite work for the epub version. Later this year I'm planning a more thorough revision that will be epub-friendly from the beginning. I have some other projects I need to finish before I can start it though, and I'm tentatively aiming to have the revision complete mid-2021

**Comment by ccamara** [comment]
Oh, thank you for the explanation and for reviewing the content.
